querybook/webapp/ui/Popover/Popover.tsx

```diff
@@ -146,6 +150,11 @@ export const PopoverContainer = React.forwardRef<
                 } else if (subLayout === 'right') {
                     newWrapperStyle.left =
                         box.left + box.width - contentBox.width;
+                    // keep the popover on screen
+                    if (newWrapperStyle.left < 0) {
+                        newWrapperStyle.left = box.left;
+                    }
```

Comment:
could be just `newWrapperStyle.left = 0`?

or 
`newWrapperStyle.left = Math.max(box.left + box.width - contentBox.width, 0);`

Reply:
Also does it also need to check the right side as well for `subLayout === 'left'` above and the else below?